terraform/spec/fixtures/projects/terraform_providers_namespace/main.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_providers {
+    fastly = {
+      source = "terraform-providers/fastly"
```

Comment:
I wonder if this is a legacy mapping that only works with some set of old providers?

Reply:
It's also not a shorthand for `hashicorp` as this does not work:

```
terraform {
  required_providers {
    consul = {
      source = "terraform-providers/consul"
    }
  }
}
```

Although, `source = "hashicorp/consul"` and `source = "hashicorp/consul"` works.

Reply:
Found this reference, looks like a legacy thing https://github.com/hashicorp/terraform/blob/d35bc0531255b496beb5d932f185cbcdb2d61a99/internal/getproviders/didyoumean_test.go#L48-L54